json_schema_fuzz.py

```diff
@@ -43,6 +44,10 @@ def generate_json(schema):
     elif type == "boolean":
         return random_boolean(schema)
     elif type == "string":
-        return random_string(schema)
+        pattern = schema.get("pattern", None) # check if string is restricted to a regex pattern
```

Comment:
My feeling is that this logic should go inside `random_string()`.

Reply:
Yep! I realized that as well!